packages/bazel-bot/docker-image/docker-main.sh

```diff
@@ -18,10 +18,27 @@ set -e
 git config --global user.email "bazel-bot-development[bot]@users.noreply.github.com"
 git config --global user.name "Bazel Bot"
 
-git clone https://github.com/googleapis/googleapis.git
-git clone https://github.com/googleapis/googleapis-gen.git
+SOURCE_CLONE_ARGS=
+if [[ $SOURCE_BRANCH != "" ]]
+then
+  SOURCE_CLONE_ARGS="-b $SOURCE_BRANCH"
+fi
+
+SOURCE_CLONE_ARGS=
```

Comment:
Lines 27-31 look identical to lines 21-25.  Is that intended?

Reply:
How weird - I must have copy/pasted twice. Thanks for spotting - will remove one copy

Reply:
Fixed now, thanks.